test/__snapshot__/transformers/es2016/abstract.md

```diff
@@ -0,0 +1,143 @@
+# Kataw parser test case
+
+## Options
```

Comment:
I got the feeling you are overloading the .md file and hard to read the important stuff. That's why I suggested - and also have it like that locale for the printer - separate .md for parser and printer. 
That way the options would exist only once - not 3x as here. Looks messy

Reply:
@aladdin-add This is how my "printer.md" looks like locale. Clean.

![printer_md](https://user-images.githubusercontent.com/31855118/112099164-ff2b6780-8bdd-11eb-8eff-4d959a5f71a4.png)

Reply:
I think it's just you want -- if you just put the transform options.

Reply:
removed the printer options: https://github.com/kataw/kataw/blob/35cfd537e32b5807d9b0646262edb5df9076ca74/test/__snapshot__/transformers/es2016/abstract.md